### PR TITLE
Market

### PR DESCRIPTION
**Pull request by fdominguezclaro**
Agregué la tab Market a tabs.html para acceder a la pestaña de Market. El icono más correcto me pareció "md-basket".

Se creó el componente market con un mockup de su estructura inicial. Por mientras los puntos de karma están siendo accedido mediante el Storage.
Estructura de la página de Market:

- 2 cards con la cantidad de órdenes de compra y venta
- 2 botones para poder comprar o vender. Estos debieran estar activados dependiendo si tiene almuerzo para vender o no.
- Una card inferior para mostrar el karma actual.

img_0637

### CODE REVIEW

File: src/pages/tabs/tabs.ts [hunk 1]
```diff
@@ -12,6 +14,7 @@ export class TabsPage {
   tab1Root = HomePage;
   tab2Root = AboutPage;
   tab3Root = ContactPage;
+  tab4Root = MarketPage;
```
Comment (monkeyci) [inline src/pages/tabs/tabs.ts]: The class property 'tab4Root' must be marked either 'private', 'public', or 'protected'<br>expected member-variable-declaration: 'tab4Root' to have a typedef

File: src/pages/market/market.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { NavController } from 'ionic-angular';
+import { Storage } from "@capacitor/core";
+
+@Component({
+  selector: 'page-market',
+  templateUrl: 'market.html'
```
Comment (monkeyci) [inline src/pages/market/market.ts]: Missing trailing comma

File: src/pages/market/market.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+})
+export class MarketPage {
+  karma: string
+
+  constructor(public navCtrl: NavController) {
+    Storage.get({key: 'karma'}).then((resp) => {
+      this.karma = resp.value
```
Comment (monkeyci) [inline src/pages/market/market.ts]: Missing semicolon

File: src/pages/market/market.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    })
```
Comment (monkeyci) [inline src/pages/market/market.ts]: Missing semicolon

File: src/pages/market/market.ts [hunk 5]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { NavController } from 'ionic-angular';
+import { Storage } from "@capacitor/core";
```
Comment (monkeyci) [inline src/pages/market/market.ts]: " should be '

File: src/pages/market/market.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  constructor(public navCtrl: NavController) {
+    Storage.get({key: 'karma'}).then((resp) => {
+      this.karma = resp.value
+    })
+  }
+
+  sell(){
+  }
+
+  buy(){
```
Comment (monkeyci) [inline src/pages/market/market.ts]: The class method 'buy' must be marked either 'private', 'public', or 'protected'<br>block is empty<br>missing whitespace

File: src/pages/market/market.ts [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+export class MarketPage {
+  karma: string
+
+  constructor(public navCtrl: NavController) {
+    Storage.get({key: 'karma'}).then((resp) => {
+      this.karma = resp.value
+    })
+  }
+
+  sell(){
```
Comment (monkeyci) [inline src/pages/market/market.ts]: The class method 'sell' must be marked either 'private', 'public', or 'protected'<br>block is empty<br>missing whitespace

File: src/pages/market/market.ts [hunk 8]
```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import { NavController } from 'ionic-angular';
+import { Storage } from "@capacitor/core";
+
+@Component({
+  selector: 'page-market',
+  templateUrl: 'market.html'
+})
+export class MarketPage {
+  karma: string
```
Comment (monkeyci) [inline src/pages/market/market.ts]: The class property 'karma' must be marked either 'private', 'public', or 'protected'<br>Missing semicolon

File: src/pages/market/market.scss [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+      bottom: 2px;
+    }
+
+    .orden{
+      height: 75px;
+    }
+    .topmargin{
+      margin-top:30px;
+  }
+
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Unexpected whitespace at end of line   no-eol-whitespace

File: src/pages/market/market.scss [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+    .bottom{
+      position: relative;
+      bottom: 2px;
+    }
+
+    .orden{
+      height: 75px;
+    }
+    .topmargin{
+      margin-top:30px;
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 4 spaces       indentation<br>Expected single space after ":"        declaration-colon-space-after

File: src/pages/tabs/tabs.ts [hunk 11]
```diff
@@ -3,15 +3,18 @@ import { Component } from '@angular/core';
...
 })
 export class TabsPage {
 
-  tab1Root = HomePage;
-  tab2Root = AboutPage;
-  tab3Root = ContactPage;
+  tab1Root: Function = HomePage;
+  tab2Root: Function = AboutPage;
+  tab3Root: Function = ContactPage;
+  tab4Root: Function = MarketPage;
```
Comment (monkeyci) [inline src/pages/tabs/tabs.ts]: The class property 'tab4Root' must be marked either 'private', 'public', or 'protected'

File: src/pages/tabs/tabs.ts [hunk 12]
```diff
@@ -3,15 +3,18 @@ import { Component } from '@angular/core';
...
   templateUrl: 'tabs.html'
 })
 export class TabsPage {
 
-  tab1Root = HomePage;
-  tab2Root = AboutPage;
-  tab3Root = ContactPage;
+  tab1Root: Function = HomePage;
+  tab2Root: Function = AboutPage;
+  tab3Root: Function = ContactPage;
```
Comment (monkeyci) [inline src/pages/tabs/tabs.ts]: The class property 'tab3Root' must be marked either 'private', 'public', or 'protected'

File: src/pages/tabs/tabs.ts [hunk 13]
```diff
@@ -3,15 +3,18 @@ import { Component } from '@angular/core';
...
 @Component({
   templateUrl: 'tabs.html'
 })
 export class TabsPage {
 
-  tab1Root = HomePage;
-  tab2Root = AboutPage;
-  tab3Root = ContactPage;
+  tab1Root: Function = HomePage;
+  tab2Root: Function = AboutPage;
```
Comment (monkeyci) [inline src/pages/tabs/tabs.ts]: The class property 'tab2Root' must be marked either 'private', 'public', or 'protected'

File: src/pages/tabs/tabs.ts [hunk 14]
```diff
@@ -3,15 +3,18 @@ import { Component } from '@angular/core';
...
 
 @Component({
   templateUrl: 'tabs.html'
 })
 export class TabsPage {
 
-  tab1Root = HomePage;
-  tab2Root = AboutPage;
-  tab3Root = ContactPage;
+  tab1Root: Function = HomePage;
```
Comment (monkeyci) [inline src/pages/tabs/tabs.ts]: The class property 'tab1Root' must be marked either 'private', 'public', or 'protected'

File: src/pages/market/market.ts [hunk 15]
```diff
@@ -0,0 +1,26 @@
...
+      this.karma = resp.value;
+    });
+  }
+
+  private sell() {
+    undefined
+  }
+
+  private buy() {
+    undefined
```
Comment (monkeyci) [inline src/pages/market/market.ts]: unused expression, expected an assignment or function call<br>Missing semicolon

File: src/pages/market/market.ts [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+  karma: string
+
+  constructor(public navCtrl: NavController) {
+    Storage.get({key: 'karma'}).then((resp) => {
+      this.karma = resp.value;
+    });
+  }
+
+  private sell() {
+    undefined
```
Comment (monkeyci) [inline src/pages/market/market.ts]: unused expression, expected an assignment or function call<br>Missing semicolon

File: src/pages/market/market.scss [hunk 17]
```diff
@@ -0,0 +1,20 @@
...
+    }
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 2 spaces    indentation

File: src/pages/market/market.scss [hunk 18]
```diff
@@ -0,0 +1,20 @@
...
+    .bottom{
+      position: relative;
+      bottom: 2px;
+    }
+
+    .orden{
+      height: 75px;
+    }
+    .topmargin{
+      margin-top:30px;
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 4 spaces    indentation<br>Expected single space after ":"     declaration-colon-space-after

File: src/pages/market/market.scss [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+
+    .bottom{
+      position: relative;
+      bottom: 2px;
+    }
+
+    .orden{
+      height: 75px;
+    }
+    .topmargin{
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected empty line before rule     rule-empty-line-before<br>Expected indentation of 2 spaces    indentation<br>Expected single space before "{"    block-opening-brace-space-before

**Comment by fdominguezclaro** [comment]
[Agregar en aplicación móvil tab vacío con nombre "Mercado"](https://trello.com/c/htU29kRU/13-agregar-en-aplicaci%C3%B3n-m%C3%B3vil-tab-vac%C3%ADo-con-nombre-mercado)

**Comment by fdominguezclaro** [comment]
[Maqueta de pantalla de mercado](https://trello.com/c/ucxshfnV/14-maqueta-de-pantalla-de-mercado)

File: src/pages/market/market.scss [hunk 20]
```diff
@@ -0,0 +1,20 @@
+page-market {
+
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center'
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected a trailing semicolon       declaration-block-trailing-semicolon

File: src/pages/market/market.scss [hunk 21]
```diff
@@ -0,0 +1,20 @@
+page-market {
+
+  ion-card {
```
Comment (monkeyci) [inline src/pages/market/market.scss:2]: Unexpected empty line before rule   rule-empty-line-before

File: src/pages/login/login.ts [hunk 22]
```diff
@@ -81,6 +81,7 @@ export class LoginPage {
           this.navCtrl.push(TabsPage)
         },
         err => {
+          console.log(err)
```
Comment (monkeyci) [inline src/pages/login/login.ts]: Calls to 'console.log' are not allowed.<br>Missing semicolon

File: src/app/app.module.ts [hunk 23]
```diff
@@ -34,7 +36,8 @@ import { SplashScreen } from '@ionic-native/splash-screen';
     ContactPage,
     HomePage,
     TabsPage,
-    LoginPage
+    LoginPage,
+    MarketPage
```
Comment (monkeyci) [inline src/app/app.module.ts]: Missing trailing comma

File: src/app/app.module.ts [hunk 24]
```diff
@@ -21,7 +22,8 @@ import { SplashScreen } from '@ionic-native/splash-screen';
...
```
Comment (monkeyci) [inline src/app/app.module.ts]: Missing trailing comma

File: src/pages/market/market.scss [hunk 25]
```diff
@@ -0,0 +1,19 @@
...
+
+  .bottom {
+    position: relative;
+    bottom: 2px;
+  }
+
+  .orden {
+    height: 75px;
+  }
+  .topmargin {
```
Comment (monkeyci) [inline src/pages/market/market.scss:12]: Expected empty line before rule   rule-empty-line-before

File: src/pages/market/market.scss [hunk 26]
```diff
@@ -0,0 +1,20 @@
...
+  .bottom {
+    position: relative;
+    bottom: 2px;
+  }
+
+  .orden {
+    height: 75px;
+  }
+  
+  .topmargin {
```
Comment (monkeyci) [inline src/pages/market/market.scss:12]: Expected empty line before rule        rule-empty-line-before

Comment (monkeyci) [inline src/pages/market/market.scss:12]: Expected empty line before rule        rule-empty-line-before

**Comment by fdominguezclaro** [comment]
Hice un par de fix anidados, cuando me di cuenta volví al hash correcto...
Puede ser que Hound se quede pegado?  Los errores que me tira ahora según yo los tengo resueltos...

File: src/pages/market/market.html [hunk 27]
```diff
@@ -0,0 +1,51 @@
...
+      Market
+    </ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content padding>
+  <ion-grid>
+    <ion-card class='orden'>
+      <ion-card-header>
+        Ordenes de compra:
```
Comment (bunzli) [inline src/pages/market/market.html]: Órdenes 🐸

**Comment by fdominguezclaro** [comment]
[Que a través de la API pueda obtener una lista de las ordenes de venta de almuerzo](https://trello.com/c/8A8FmhvP/17-que-a-trav%C3%A9s-de-la-api-pueda-obtener-una-lista-de-las-ordenes-de-venta-de-almuerzo)

**Comment by fdominguezclaro** [comment]
[Que se pueda crear nueva orden limite de venta via API](https://trello.com/c/Q8Zv0LlU/18-que-se-pueda-crear-nueva-orden-limite-de-venta-via-api)

**Comment by fdominguezclaro** [comment]
[Que se pueda comprar almuerzo agregando una market order via API](https://trello.com/c/ZeepdRfb/19-que-se-pueda-comprar-almuerzo-agregando-una-market-order-via-api)

File: src/pages/market/market.scss [hunk 28]
```diff
@@ -0,0 +1,21 @@
...
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+    .bottom{
+      position: relative;
+      bottom: 2px;
+    }
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 2 spaces   indentation

File: src/pages/market/market.scss [hunk 29]
```diff
@@ -0,0 +1,21 @@
+page-market {
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+    .bottom{
+      position: relative;
+      bottom: 2px;
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 4 spaces   indentation

File: src/pages/market/market.scss [hunk 30]
```diff
@@ -0,0 +1,21 @@
+page-market {
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+    .bottom{
+      position: relative;
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 4 spaces   indentation

File: src/pages/market/market.scss [hunk 31]
```diff
@@ -0,0 +1,21 @@
+page-market {
+  ion-card {
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+    .bottom{
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Expected indentation of 2 spaces   indentation<br>Expected single space before "{"   block-opening-brace-space-before

**Review by gmq — APPROVED** [review]

File: src/pages/login/login.ts [hunk 32]
```diff
@@ -78,7 +78,7 @@ export class LoginPage {
         response => {
           const karma = JSON.parse(response.data).karma
           this.storeInitialData(code, karma)
-          this.navCtrl.push(TabsPage)
+          this.navCtrl.push(HomePage)
```
Comment (monkeyci) [inline src/pages/login/login.ts]: Missing semicolon

File: src/pages/login/login.ts [hunk 33]
```diff
@@ -32,7 +32,7 @@ export class LoginPage {
     this.checkHamCode().then((result) => {
       const hamcode = result.value
       if(hamcode && parseInt(hamcode) >= 0){
-        this.navCtrl.push(TabsPage)
+        this.navCtrl.push(HomePage)
```
Comment (monkeyci) [inline src/pages/login/login.ts]: Missing semicolon

File: src/pages/home/home.ts [hunk 34]
```diff
@@ -59,6 +60,7 @@ export class HomePage {
       response => {
         const lunchers = JSON.parse(response.data).winning_lunchers
         this.lunchers = lunchers
+        this.lunchersAmount = lunchers.length
```
Comment (monkeyci) [inline src/pages/home/home.ts]: Missing semicolon

File: src/pages/home/home.ts [hunk 35]
```diff
@@ -11,6 +11,7 @@ export class HomePage {
   karma: string
   hamcode: string
   lunchers: any
+  lunchersAmount: any
```
Comment (monkeyci) [inline src/pages/home/home.ts]: The class property 'lunchersAmount' must be marked either 'private', 'public', or 'protected'<br>Missing semicolon

File: src/pages/home/home.scss [hunk 36]
```diff
@@ -1,3 +1,19 @@
...
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
+  ion-item{
+    border-color: white;
+    &.stripped{
+      background: repeating-linear-gradient(45deg, #F6FAFC, #F6FAFC 15px, white 15px, white 30px);
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected "#F6FAFC" to be "#f6fafc"    color-hex-case<br>Unexpected named color "white"        color-named

File: src/pages/home/home.scss [hunk 37]
```diff
@@ -1,3 +1,19 @@
...
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
+  ion-item{
+    border-color: white;
+    &.stripped{
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected empty line before rule       rule-empty-line-before<br>Expected single space before "{"      block-opening-brace-space-before<br>Unexpected qualifying type selector   selector-no-qualifying-type

File: src/pages/home/home.scss [hunk 38]
```diff
@@ -1,3 +1,19 @@
...
+    margin-top: 10pt;
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
+  ion-item{
+    border-color: white;
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Unexpected named color "white"        color-named

File: src/pages/home/home.scss [hunk 39]
```diff
@@ -1,3 +1,19 @@
...
+    font-size: 18pt;
+    margin-top: 10pt;
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
+  ion-item{
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected empty line before rule       rule-empty-line-before<br>Expected single space before "{"      block-opening-brace-space-before

File: src/pages/home/home.scss [hunk 40]
```diff
@@ -1,3 +1,20 @@
...
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
 
+  ion-item {
+    border-color: #ffffff;
+    &.stripped {
+      background: repeating-linear-gradient(45deg, #f6fafc, #f6fafc 15px, #ffffff 15px, #ffffff 30px);
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected "#ffffff" to be "#fff"       color-hex-length

File: src/pages/home/home.scss [hunk 41]
```diff
@@ -1,3 +1,20 @@
...
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
+      height: 28pt;
+      width: 28pt;
+    }
+  }
 
+  ion-item {
+    border-color: #ffffff;
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected "#ffffff" to be "#fff"       color-hex-length

File: src/pages/home/home.scss [hunk 42]
```diff
@@ -1,3 +1,20 @@
-page-home {
+.page-home {
+  background-color: #DCEBFD;
+  .header {
+    font-size: 18pt;
+    margin-top: 10pt;
+    margin-bottom: 10pt;
+    padding: 0 10pt;
+    img {
```
Comment (monkeyci) [inline src/pages/home/home.scss:10]: Expected empty line before rule       rule-empty-line-before

File: src/pages/home/home.scss [hunk 43]
```diff
@@ -1,3 +1,20 @@
-page-home {
+.page-home {
+  background-color: #DCEBFD;
+  .header {
```
Comment (monkeyci) [inline src/pages/home/home.scss:4]: Expected empty line before rule       rule-empty-line-before

File: src/pages/home/home.scss [hunk 44]
```diff
@@ -1,3 +1,20 @@
-page-home {
+.page-home {
+  background-color: #DCEBFD;
```
Comment (monkeyci) [inline src/pages/home/home.scss]: Expected "#DCEBFD" to be "#dcebfd"    color-hex-case

File: src/pages/home/home.ts [hunk 45]
```diff
@@ -107,16 +109,16 @@ export class HomePage {
...
+            this.howManyWantToLunch(hamcode);
           } else {
-            this.showToast("Error :( ")
+            this.showToast("Error :( ");
           }
         },
         err => {
-          const errorMessage = JSON.parse(err.error).message
-          this.showToast(errorMessage)
+          const errorMessage = JSON.parse(err.error).message;
```
Comment (monkeyci) [inline src/pages/home/home.ts:120]: expected variable-declaration: 'errorMessage' to have a typedef

File: src/pages/home/home.ts [hunk 46]
```diff
@@ -107,16 +109,16 @@ export class HomePage {
       this.http.post('http://pl-ham.herokuapp.com/current_lunchers', {}, headers)
       .then(
         response => {
-          const success = JSON.parse(response.data).success
+          const success = JSON.parse(response.data).success;
```
Comment (monkeyci) [inline src/pages/home/home.ts:112]: expected variable-declaration: 'success' to have a typedef

File: src/pages/home/home.ts [hunk 47]
```diff
@@ -86,19 +88,19 @@ export class HomePage {
...
+        const errorMessage = JSON.parse(err.error).message;
+        this.showToast(errorMessage);
       }
     )
   }
 
   willLunch(){
     Storage.get({key: 'hamcode'}).then((resp) => {
-      const hamcode = resp.value
+      const hamcode = resp.value;
```
Comment (monkeyci) [inline src/pages/home/home.ts:103]: expected variable-declaration: 'hamcode' to have a typedef

File: src/pages/home/home.ts [hunk 48]
```diff
@@ -86,19 +88,19 @@ export class HomePage {
...
       response => {
-        const wantToLunch = JSON.parse(response.data).current_lunchers.length
-        this.showToast('Hmm.. Ya van ' + wantToLunch + ' que les gustaría almorzar aquí hoy.')
+        const wantToLunch = JSON.parse(response.data).current_lunchers.length;
+        this.showToast('Hmm.. Ya van ' + wantToLunch + ' que les gustaría almorzar aquí hoy.');
       },
       err => {
-        const errorMessage = JSON.parse(err.error).message
-        this.showToast(errorMessage)
+        const errorMessage = JSON.parse(err.error).message;
```
Comment (monkeyci) [inline src/pages/home/home.ts:95]: expected variable-declaration: 'errorMessage' to have a typedef

File: src/pages/home/home.ts [hunk 49]
```diff
@@ -86,19 +88,19 @@ export class HomePage {
     this.http.get('http://pl-ham.herokuapp.com/current_lunchers', {}, headers)
     .then(
       response => {
-        const wantToLunch = JSON.parse(response.data).current_lunchers.length
-        this.showToast('Hmm.. Ya van ' + wantToLunch + ' que les gustaría almorzar aquí hoy.')
+        const wantToLunch = JSON.parse(response.data).current_lunchers.length;
```
Comment (monkeyci) [inline src/pages/home/home.ts:91]: expected variable-declaration: 'wantToLunch' to have a typedef

File: src/pages/home/home.ts [hunk 50]
```diff
@@ -57,12 +58,13 @@ export class HomePage {
...
-        const lunchers = JSON.parse(response.data).winning_lunchers
-        this.lunchers = lunchers
+        const lunchers = JSON.parse(response.data).winning_lunchers;
+        this.lunchers = lunchers;
+        this.lunchersAmount = lunchers.length;
       },
       err => {
-        const errorMessage = JSON.parse(err.error).message
-        this.showToast(errorMessage)
+        const errorMessage = JSON.parse(err.error).message;
```
Comment (monkeyci) [inline src/pages/home/home.ts:66]: expected variable-declaration: 'errorMessage' to have a typedef

File: src/pages/home/home.ts [hunk 51]
```diff
@@ -57,12 +58,13 @@ export class HomePage {
     this.http.get('http://pl-ham.herokuapp.com/winning_lunchers', {}, headers)
     .then(
       response => {
-        const lunchers = JSON.parse(response.data).winning_lunchers
-        this.lunchers = lunchers
+        const lunchers = JSON.parse(response.data).winning_lunchers;
```
Comment (monkeyci) [inline src/pages/home/home.ts:61]: expected variable-declaration: 'lunchers' to have a typedef

File: src/pages/home/home.ts [hunk 52]
```diff
@@ -33,16 +34,16 @@ export class HomePage {
...
+        this.karma = karma;
         Storage.set({
           key: 'karma',
           value: karma
         })
       },
       err => {
-        const errorMessage = JSON.parse(err.error).message
-        this.showToast(errorMessage)
+        const errorMessage = JSON.parse(err.error).message;
```
Comment (monkeyci) [inline src/pages/home/home.ts:45]: expected variable-declaration: 'errorMessage' to have a typedef

File: src/pages/market/market.scss [hunk 53]
```diff
@@ -0,0 +1,19 @@
...
+    text-align: center;
+    justify-content: 'center';
+    align-items: 'center';
+  }
+
+  .orden {
+    height: 75px;
+  }
+
+  .orden {
```
Comment (monkeyci) [inline src/pages/market/market.scss]: Unexpected duplicate selector "page-market .orden", first used at line 8   no-duplicate-selectors

**Pull request merged**